chromadb/api/local.py

```diff
@@ -24,6 +24,7 @@
 from chromadb.telemetry import Telemetry
```

Comment:
The primary change in this file is to add persist() to all the write path methods. I would love verification that none are missed.

Reply:
Update: no longer relevant since we don't persist()